apis/hive/v1/hiveconfig_types.go

```diff
@@ -173,6 +177,17 @@ type ReleaseImageVerificationConfigMapReference struct {
 	Name string `json:"name"`
 }
 
+// PrivateLinkConfig defines the configuration for the privatelink controller.
+type PrivateLinkConfig struct {
+	// AWS is the configuration for AWS hub and link resources.
+	// +optional
+	AWS *AWSPrivateLinkConfig `json:"aws,omitempty"`
```

Comment:
As discussed, let's remove this for the MVP and reintroduce it later when we're ready to deprecate the old one.

Reply:
[Done](https://github.com/openshift/hive/compare/b8f6a27707df3dd2dcd91175858fef744a0bc2f7..347334eb55af63da144fd48ed037f20948c499cf)